docs/api.md

```diff
@@ -644,6 +644,33 @@ most.from([1, 2])
 	.observe(console.log.bind(console));
 ```
 
+### flatMapEnd
+
+####`stream.flatMapEnd(f) -> Stream`
+####`most.flatMapEnd(f, stream) -> Stream`
+
+Map the end event to a new stream, and begin emitting its values.  Note that `f` *must* return a stream.
```

Comment:
I've been trying to avoid imperative terminology like "begin emitting", in favor of declarative terminology.  Maybe we can describe this as a form of concatenation.  IOW, conceptually, `flatMapEnd` concatenates `stream` and `f(end)` (even though the value of `end` can't be known until the future when it occurs!).  Thoughts?